src/control/lib/telemetry/telemetry.go

```diff
@@ -50,7 +51,7 @@ const (
 	BadIntVal   = int64(BadUintVal >> 1)
 	BadDuration = time.Duration(BadIntVal)
 
-	pathSep = '/'
+	PathSep = filepath.Separator
```

Comment:
IMO this line should be reverted. `filepath.Separator` is a platform-specific value. The telemetry shmem region isn't a true filesystem, and the separator is always '/' regardless of platform.

Reply:
The thought here was that switching to `filepath.Separator` would allow us to use the `filepath` functions regardless of platform. As exported from the engine, the metrics are agnostic. We only define the separator at this level. I don't realistically anticipate that we'll be running DAOS on windows or VMS or whatever, but this seems less arbitrary.

Reply:
Ah, you're right. I was thinking we used the engine-defined paths somewhere in the control plane.